2 Business layer/CandidateEvaluator.Core/Handlers/Commands/UpdateCategoryHandler.cs

```diff
@@ -0,0 +1,30 @@
+﻿using CandidateEvaluator.Contract.Commands.Category;
+using CandidateEvaluator.Contract.Handlers;
+using CandidateEvaluator.Contract.Models;
+using CandidateEvaluator.Contract.Services;
+using System;
+using System.Threading.Tasks;
+
+namespace CandidateEvaluator.Core.Handlers.Commands
+{
+    public class UpdateCategoryHandler : ICommandHandler<UpdateCategory>
+    {
+        private readonly ICategoryService _categoryService;
+
+        public UpdateCategoryHandler(ICategoryService categoryService)
+        {
+            _categoryService = categoryService;
```

Comment:
I believe that handlers should operate straight on repositories/context. The service layer is redundant here.